src/points.jl

```diff
@@ -30,6 +31,7 @@ immutable LatLon{T <: Number}
     lat::T
     lon::T
 end
+LatLon(lat, lon) = LatLon(promote(lat, lon)...)
```

Comment:
@omus This overwrote the `LLA(x, datum)` def below (line 37), which is a method @andyferris implemented. Havent'